### PR TITLE
Bump version 4.1.0

### PR DESCRIPTION
**Pull request by planktonic**
The following changes are contained in this pull request:

- Bumps version
- updates changelog

### CODE REVIEW

**Review by klappradla — APPROVED** [review]
🪨

**Comment by klappradla** [comment]
Happy release 🚀

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## [[4.1.0]](https://github.com/bitcrowd/bitstyles/releases/tag/v4.1.0) - 2021-12-20
```
[the remark this answers is not in the document]
Comment (planktonic) [reply CHANGELOG.md:3]: Good point! I’d add it to the CONTRIBUTING.md file in #627, and in the `Updating` section in the same PR (https://github.com/bitcrowd/bitstyles/pull/627/files#diff-70392291b42db79b2c700adf64716d6a275561ae135dbf6b4bbb84f2325396dbR21)

We don’t follow it exactly though — we have a `breaking` section. Maybe we should move `breaking` changes into `changed`?

[the remark this answers is not in the document]
Comment (klappradla) [reply CHANGELOG.md:3]: Well, "keepachangelog.com" is not a standard, just a place to start. I think "breaking" is a great fit - would keep it 💯

**Pull request merged**